### PR TITLE
adding navfn and planner server param descriptions

### PR DESCRIPTION
**Pull request by SteveMacenski**
@Marwan99 please take a look

### CODE REVIEW

**Comment by SteveMacenski** [comment]
#21

File: configuration/packages/configuring-navfn.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. _configuring_navfn:
+
+NavFn Planner
+#############
+
+Source code on Github_.
+
+.. _Github: https://github.com/ros-planning/navigation2/tree/master/nav2_navfn_planner
+
+The Navfn Planner plugin implements a wavefront Dij. or A* expanded planner.
```
Comment (Marwan99) [inline configuration/packages/configuring-navfn.rst:10]: Dijkstra instead of Dij.?

File: configuration/packages/configuring-navfn.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+:``<name>``.use_astar:
+
+  ==== =======
+  Type Default                                                   
+  ---- -------
+  bool False            
+  ==== =======
+
+  Description
+    Whether to use A*, if false, uses Dijstra's expansion
```
Comment (Marwan99) [inline configuration/packages/configuring-navfn.rst:37]:
```suggestion
    Whether to use A*. If false, uses Dijkstra's expansion.
```

File: configuration/packages/configuring-navfn.rst (hunk 1 above)
Comment (SteveMacenski) [inline configuration/packages/configuring-navfn.rst:10]:
```suggestion
The Navfn Planner plugin implements a wavefront Dijkstra or A* expanded planner.
```

**Pull request merged**